src/util/nft.js

```diff
@@ -175,6 +176,12 @@ export function checkIsWritingNFT(classMetadata) {
   );
 }
 
+export function checkIsWritingNFTByNFTId(id) {
+  return /^writing-[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(
```

Comment:
should add comment stating this is kind of performance hack, since wnft should be judged from class metadata

Reply:
Added